docs/adr/0002-videos-languages.rst

```diff
@@ -0,0 +1,64 @@
+ADR 0002 - Videos languages
+===========================
+
+Status
+------
+
+Proposed
+
+
+Context
+-------
+
+We want to think Marsha as accessible from the beginning. At least from the point of view of the videos, which are the main content available.
+
+We can think about a video as a "set of related videos with their accessibility counterparts".
+
+Counterparts
+~~~~~~~~~~~~
+
+Audio
++++++
+
+The same video can be proposed with different audio tracks. Which often result in many videos. That's our case, we don't manage audio tracks separately.
+
+Subtitles
++++++++++
+
+In addition to audio tracks, many subtitles tracks can be available.
+
+Sign language
++++++++++++++
+
+Some people with disabilities could want a video with the sign language transcript. For this it can be a video incorporated in the original one, or an other video displayed on the site.
+
+As sign languages are not the same for every spoken language, it can exist many sign languages videos for a single video.
+
+Summary
+~~~~~~~
+
+What we call a "video" can in fact have many videos, each one with a different audio-track. Each of these video can have different subtitles, and different sign-languages videos.
+
+
+Decision
+--------
+
+We decided to take all this elements into account right from the beginning.
+
+We'll use `django-parler <https://github.com/django-parler/django-parler>`_ to manage translations.
+
+So we have a main Django model named ``Video``, from an author. As a video can be in many languages (audio tracks), we use ``django-parler`` for this: the title and the video url are managed this way.
```

Comment:
Maybe we need an explicit translation model for the video so that the we can point the foreignkey of AudioTrack and SubtitleTrack objects to djangoparler's video translation model and not to the Video model.

Reply:
After what we discussed yesterday, I think we don't need Django Parler for this. 
videotracks, audiotracks and subtitle tracks can be handled the same way with a foreign key to the main video object. It's what you have in mind right?

Reply:
We have a deal !